crates/router/src/core/payments/helpers.rs

```diff
@@ -1236,15 +1236,16 @@ pub fn create_authorize_url(
 pub fn create_webhook_url(
     router_base_url: &String,
     merchant_id: &id_type::MerchantId,
-    connector_name: impl std::fmt::Display,
+    merchant_connector_id: &id_type::MerchantConnectorAccountId,
 ) -> String {
     format!(
-        "{}/webhooks/{}/{}",
+        "/webhooks/{}/{}",
```

Comment:
the base url should be in the beginning, and there are only two params in the format string, whereas there are three parameters